compiler/circle-mpqsolver/src/Methods/BisectionSolver.cpp

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
about the folder name; `Methods` starts with capital, but `utils` doesn't.
it would be better to not mix them; maybe be use lower case.

Reply:
> about the folder name; `Methods` starts with capital, but `utils` doesn't. it would be better to not mix them; maybe be use lower case.

Ahhh... Sorry. Will be fixed.